### ISSUE TITLE
Mailhog container tag is no longer valid

### ISSUE THREAD

**Issue by goldenapples** [humanmade/altis-local-server#485]
All Travis builds on my project are currently failing when trying to fetch the Docker container specified in the docker-compose file with the following error:

```
Pulling mailhog (cd2team/mailhog:1632011321)...
manifest for cd2team/mailhog:1632011321 not found: manifest unknown: manifest unknown
Services failed to start successfully.
The command "composer local-server start" failed and exited with 1 during .
``` 

Looking at the [tags available for cd2team/mailhog](https://hub.docker.com/r/cd2team/mailhog/tags?page=1&ordering=last_updated), it looks like they might have removed all their old releases 2 days ago.

**Comment by goldenapples** [comment]
Fixed by #486.

**Comment by rmccue** [comment]
Ref https://github.com/CODESIGN2/mailhog-docker/issues/9 for posterity
